components/shotAccordion.js

```diff
@@ -37,13 +37,24 @@ function DataField(field, value) {
         </View>
       );
     case "target":
-      return (
-        <Row
-          key={field}
-          name={prettyTitle[field]}
-          value={`${numTrunc(value)} yd`}
-        />
-      );
+      switch (drillInfo.requirements[0].type) {
```

Comment:
Let's not use target here. create a new field callled "distance" so you don't have to handle these inputtedPutt edge cases everywhere